### PR TITLE
adds an example for the --update-filetypes flag

### PR DESCRIPTION
**Pull request by mandresm**
adds an example for the --update-filetypes flag

### CODE REVIEW

File: docs/esm_runscripts.rst [hunk 1]
```diff
@@ -18,21 +18,26 @@ Arguments
...
+  ``-t TASK``, ``--task TASK``                         The task to run. Choose from: ``compute``, ``post``, ``couple``, ``tidy_and_resubmit``.
+  ``-p PID``, ``--pid PID``                            The PID of the task to observe.
+  ``-x EXCLUDE``, ``--exclude EXCLUDE``                E[x]clude this step.
+  ``-o ONLY``, ``--only ONLY``                         [o]nly do this step.
+  ``-r RESUME_FROM``, ``--resume-from RESUME_FROM``    [r]esume from the specified run/step (i.e. to resume a second run you'll need to use ``-r 2``).
+  ``-U``, ``--update``                                 [U]pdate the runscript in the experiment folder and associated files
+  ``--update-filetypes UPDATE_FILETYPES [UPDATE_FILETYPES ...]``    Updates the requested files from external sources in a currently 
+                                                       ongoing simulation. For example, if you want to update the binaries 
+                                                       and the configs (namelists) in a resubmission of a experiment you can 
+                                                       do this by adding ``--update-filetypes bin config`` to your
```
Comment (nwieters) [inline docs/esm_runscripts.rst]: I have a small question. Are 'bin' and 'config' the folder names in the experiment? Does it mean that all binaries in the 'bin' folder and every file in 'config' will be updated?

**Comment by nwieters** [comment]
I just did a small formatting change for the table, since the whole table did not show up in the docs.

**Comment by nwieters** [comment]
Is it mentioned somewhere in the documentation or somehow clear what can be given as UPDATE_FILETYPES argument?

**Review by nwieters — APPROVED** [review]

File: docs/esm_runscripts.rst (hunk 1 above)
Comment (mandresm) [reply docs/esm_runscripts.rst]: Not exactly, `bin` and `config` are file categories. You decide which files fall into these categories in your yaml files. But yes, it means that all the files in these cathegories will be updated.

**Comment by mandresm** [comment]
> I just did a small formatting change for the table, since the whole table did not show up in the docs.

thanks!

**Pull request merged**